ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/CloseButton.kt

```diff
@@ -16,12 +21,20 @@ fun BoxScope.CloseButton(
     onClick: () -> Unit,
 ) {
     if (shouldDisplayDismissButton) {
-        IconButton(
-            onClick = onClick,
-            modifier = Modifier.align(Alignment.TopStart),
-            enabled = actionInProgress.not(),
+        Column(
+            Modifier.align(Alignment.TopStart),
         ) {
-            Icon(painter = painterResource(id = R.drawable.close), contentDescription = null)
+            Spacer(
+                Modifier.windowInsetsTopHeight(
```

Comment:
I guess we're only applying the margin to the close button... I think properly we should apply it to the whole content right?

Reply:
Sure, I will think about how to do that, but I think I can wrap the Content of the Box in LoadedPaywall with a Column and add the spacer there.

Reply:
Actually... that won't work because the background of the paywall won't be applied to the Spacer

```
    Column(
        modifier = Modifier
            .conditional(state.isInFullScreenMode) {
                Modifier
                    .fillMaxHeight()
                    .background(backgroundColor)
            }
            .conditional(!state.isInFullScreenMode) {
                Modifier
                    .clip(
                        RoundedCornerShape(
                            topStart = UIConstant.defaultCornerRadius,
                            topEnd = UIConstant.defaultCornerRadius,
                        ),
                    )
                    .background(backgroundColor)
            },
    ) {
            Spacer(
                Modifier.windowInsetsTopHeight(
                    WindowInsets.statusBars,
                ),
            )
            Box {
                val configuration = state.configurationWithOverriddenLocale()

                CompositionLocalProvider(
                    LocalActivity provides LocalContext.current.getActivity(),
                    LocalContext provides state.contextWithConfiguration(configuration),
                    LocalConfiguration provides configuration,
                ) {
                    TemplatePaywall(state = state, viewModel = viewModel)
                }
                CloseButton(
                    shouldDisplayDismissButton = state.shouldDisplayDismissButton,
                    actionInProgress = viewModel.actionInProgress.value,
                    onClick = viewModel::closePaywall,
                )
            }
        }
```

Do you have any other suggestion?

Reply:
Hmm I guess we could apply that both to the `CloseButton` and the `TemplatePaywall` composables? 
But if that doesn't work, I'm with what you have for now.

Reply:
That's actually not an option since it wouldn't work with template 2. Nvm then, I think it's ok as is!